notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<p>In this tutorial, you will complete the following objectives:</p><p> </p><p>- Ingest feature values from a Pandas DataFrame into a feature store.</p><p>- Read feature values from the feature store's online storage into a Pandas DataFrame.</p><p>- Batch serve feature values from your feature store into a Pandas DataFrame.</p><p> </p><p>This tutorial also shows how Vertex AI Feature Store can help you update and serve feature values for online predictions as well as fetch feature values for training while avoiding data leakage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>